deposit/src/main/java/edu/unc/lib/deposit/work/AbstractDepositJob.java

```diff
@@ -52,7 +52,10 @@
 
 	@Autowired
 	protected Repository repository;
-
+	
+	@Autowired
```

Comment:
This looks good, I'll just mention that the spring context files will need to be updated to instantiate the PremisLoggerFactory to inject here.